src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -275,9 +272,9 @@ def delete_arc_agents(release_namespace, kube_config, kube_context, configuratio
             telemetry.set_user_fault()
         telemetry.set_exception(exception=error_helm_delete.decode("ascii"), fault_type=consts.Delete_HelmRelease_Fault_Type,
                                 summary='Unable to delete helm release')
-        raise CLIError("Error occured while cleaning up arc agents. " +
-                       "Helm release deletion failed: " + error_helm_delete.decode("ascii") +
-                       " Please run 'helm delete azure-arc' to ensure that the release is deleted.")
+        raise CLIInternalError("Error occured while cleaning up arc agents. " +
```

Comment:
I hope this would not unset telemetry.set_user_fault() for 'forbidden' case

Reply:
Won't unset, as these these just wraps telemetry calls within themselves. And since CLIInternalError doesn't do anything related to user_fault, it wont override the behaviour for forbidden case.